kubera-enterprise/templates/kubera-core-ui-service.yaml

```diff
@@ -6,6 +6,6 @@ spec:
   ports:
     - name: http
       port: 9091
-      targetPort: 80
+      targetPort: 8080
```

Comment:
Why is this change?

Reply:
We are moving `kubera-core-ui` image from `nginx` to `unprivileged nginx` and bydefault unprivileged nginx listens on 8080
ref- https://github.com/mayadata-io/kubera-core-ui/pull/57

Reply:
Got it